docs/groovy-script/mods/abyssaltweaker/ritual.md

```diff
@@ -0,0 +1,158 @@
+---
+title: "Necronomicon Ritual"
+titleTemplate: "Abyssal Tweaker | CleanroomMC"
+description: "Allows setting up rituals that can be executed with various tiers of Necronomicon."
+source_code_link: "https://github.com/TeamDimensional/AbyssalTweaker/blob/master/src/main/java/com/teamdimensional/abyssaltweaker/compat/groovyscript/Ritual.java"
+---
+
+# Necronomicon Ritual (Abyssal Tweaker)
+
+## Description
+
+Allows setting up rituals that can be executed with various tiers of Necronomicon.
```

Comment:
there are a few types of ritual that are not included in this. for instance, `NecronomiconSummonRitual` to summon entities or `NecronomiconCreationRitual` to create an item without using the center slot. your current code only really handles `NecronomiconInfusionRitual`.
should either mention this or add compatibility & examples

Reply:
did not know NecronomiconCreationRitual was separate. Other rituals can only be added manually at the time, without a builder. I do not mind making the builder more flexible in what kinds of rituals it can create, will do later.